### PR TITLE
feat: 557 python template S3 To BigQuery

### PR DESCRIPTION
**Pull request by tanyarw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ Please refer to the [Dataproc Templates (Python - PySpark) README](/python) for
 * [JDBCToJDBC](/python/dataproc_templates/jdbc#1-jdbc-to-jdbc) (blogpost [link](https://medium.com/google-cloud/migrating-data-from-one-databases-into-another-via-jdbc-using-dataproc-serverless-c5336c409b18))
 * [MongoToGCS](/python/dataproc_templates/mongo#mongo-to-gcs) (blogpost [link](https://medium.com/google-cloud/exporting-data-from-mongodb-to-gcs-buckets-using-dataproc-serverless-64830fb15b51))
 * [RedshiftToGCS](/python/dataproc_templates/redshift#redshift-to-gcs) (blogpost [link](https://medium.com/google-cloud/exporting-data-from-redshift-to-gcs-using-gcp-dataproc-serverless-and-pyspark-9ab78de11405))
+* [S3ToBigQuery](/python/dataproc_templates/s3#amazon-s3-to-bigquery)
```
Comment (vanshaj-bhatia) [inline README.md:63]: Link isn't correct.

**Review by vanshaj-bhatia — CHANGES_REQUESTED** [review]

File: python/dataproc_templates/s3/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+## Arguments
+
+* `s3.bq.input.location` : Amazon S3 input location. Input location must begin with `s3a://`
+* `s3.bq.access.key` : Access key to access Amazon S3 bucket
+* `s3.bq.secret.key` : Secret key to access Amazon S3 bucket
+* `s3.bq.input.format` : Input file format in Amazon S3 bucket (one of : avro, parquet, csv, json)
+* `s3.bq.output.dataset.name` : BigQuery dataset for the output table
+* `s3.bq.output.table.name` : BigQuery output table name
+* `s3.bq.ld.temp.bucket.name` : Pre existing GCS bucket name where temporary files are staged
```
[the remark this answers is not in the document]
Comment (tanyarw) [reply python/dataproc_templates/s3/README.md:15]: Matched the parameter name with Java template. Should it be changed to `s3.bq.temp.bucket.name` ?

File: python/dataproc_templates/s3/s3_to_bigquery.py [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        secret_key: str = args[constants.S3_BQ_SECRET_KEY]
+        input_file_format: str = args[constants.S3_BQ_INPUT_FORMAT]
+        bq_dataset: str = args[constants.S3_BQ_OUTPUT_DATASET_NAME]
+        bq_table: str = args[constants.S3_BQ_OUTPUT_TABLE_NAME]
+        bq_temp_bucket: str = args[constants.S3_BQ_LD_TEMP_BUCKET_NAME]
+        output_mode: str = args[constants.S3_BQ_OUTPUT_MODE]
+
+        logger.info(
+            "Starting Amazon S3 to Bigquery spark job with parameters:\n"
+            f"{pprint.pformat(args)}"
```
[the remark this answers is not in the document]
Comment (tanyarw) [reply python/dataproc_templates/s3/s3_to_bigquery.py]: Updated

File: README.md (hunk 1 above)
Comment (vanshaj-bhatia) [reply README.md:63]: Seems alright.

File: python/test/hive/util/test_hive_ddl_extractor.py [hunk 4]
```diff
@@ -20,7 +20,6 @@
 from dataproc_templates.hive.util.hive_ddl_extractor import \
     HiveDDLExtractorTemplate
 from datetime import datetime
-from google.cloud import storage
```
Comment (vanshaj-bhatia) [inline python/test/hive/util/test_hive_ddl_extractor.py:23]: Please revert this change. As it is not relevant to this issue.

**Review by vanshaj-bhatia — APPROVED** [review]

**Comment by vanshaj-bhatia** [comment]
Work on the requested changes before merging this PR.

File: python/test/hive/util/test_hive_ddl_extractor.py (hunk 4 above)
Comment (tanyarw) [reply python/test/hive/util/test_hive_ddl_extractor.py]: Done
